.gitignore

```diff
@@ -2,3 +2,5 @@ doc/
 tests/coverage/
 vendor/
 assets/css/xhstyles.css
+/cmsimple/.sessionname
+/cmsimple/log.txt

```

Comment:
Makes sense to ignore these, but it would be better to do this in a separate commit (IMHO that's good for the 1.7 branch as well; so commit to 1.7 and merge into master).

Reply:
Done. in cmsimple-xh:master

Reply:
Thanks! We'll probably have to fix the build script, to add these files. I'll have a look at this later.

Reply:
No need to do anything, since both files will be created if not already existing. Actually, it's a great idea to remove the files from the repo and to .gitinore them to prevent commits such as fe3dec84b07bd04922d38af446cea636cb96758d.